### PR TITLE
Component dialog

### PR DESCRIPTION
**Pull request by PUndef**

### CODE REVIEW

File: src/components/Dialog/DefinetelyAgree.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+import { withTranslation } from 'react-i18next';
+import { inject } from 'mobx-react';
+import PropTypes from 'prop-types';
+import Dialog from './Dialog';
+import { VerifyIcon } from '../Icons';
+// import FinPassForm from '../../stores/FormsStore/FinPassForm';
+
+// TODO Finalize after form universal create
+@inject('dialogStore')
+@withTranslation()
```
Comment (masteralx) [inline src/components/Dialog/DefinetelyAgree.js]: withTranslation upper

File: src/components/Dialog/AgreedMessage.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
+import React from 'react';
```
Comment (masteralx) [inline src/components/Dialog/AgreedMessage.js]: - Move  icon from dialog
- Create message component
- Remove Agree Dialog, reject Dialog

File: src/components/Dialog/DefaultDialogFooter.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
+import React from 'react';
```
Comment (masteralx) [inline src/components/Dialog/DefaultDialogFooter.js:1]: Check is it used anywhere

File: src/components/Dialog/TokenTransferSuccess.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
+import React from 'react';
```
Comment (masteralx) [inline src/components/Dialog/TokenTransferSuccess.js]: Separate message

File: src/stores/DialogStore/DialogItemModel.js [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+  onClose: null,
+  history: true,
+};
+
+class DialogItem {
+  constructor(name, options) {
+    this._name = name;
+    this.options = { ...defaults, ...options };
+  }
+
```
Comment (masteralx) [inline src/stores/DialogStore/DialogItemModel.js:24]: add jsdoc comments

File: src/stores/DialogStore/DialogStore.js [hunk 6]
```diff
@@ -0,0 +1,109 @@
+import {
```
Comment (masteralx) [inline src/stores/DialogStore/DialogStore.js:1]: jsdoc comments

**Review by masteralx — CHANGES_REQUESTED** [review]

File: src/components/Dialog/DefinetelyAgree.js (hunk 1 above)
Comment (PUndef) [reply src/components/Dialog/DefinetelyAgree.js]: fixed

File: src/components/Dialog/AgreedMessage.js (hunk 2 above)
Comment (PUndef) [reply src/components/Dialog/AgreedMessage.js]: fixed

File: src/components/Dialog/DefaultDialogFooter.js (hunk 3 above)
Comment (PUndef) [reply src/components/Dialog/DefaultDialogFooter.js:1]: it's used in basic dialog by default

File: src/components/Dialog/TokenTransferSuccess.js (hunk 4 above)
Comment (PUndef) [reply src/components/Dialog/TokenTransferSuccess.js]: fixed

File: src/stores/DialogStore/DialogStore.js (hunk 6 above)
Comment (PUndef) [reply src/stores/DialogStore/DialogStore.js:1]: fixed

File: src/stores/DialogStore/DialogStore.js [hunk 7]
```diff
@@ -0,0 +1,163 @@
...
+     */
+    doesExist(dialog) {
+      return this.getDialog(dialog) !== undefined;
+    }
+
+    /**
+     * Method for adding new dialog in list
+     *
+     * @param {string} name name new dialog
+     * @param {object} options options for new dialog
```
Comment (masteralx) [inline src/stores/DialogStore/DialogStore.js:63]: specify object

File: src/stores/DialogStore/DialogItemModel.js [hunk 8]
```diff
@@ -0,0 +1,77 @@
...
+  onOpen: null,
+  onClose: null,
+  history: true,
+};
+
+/**
+ * Single modal dialog model
+ */
+class DialogItem {
+  constructor(name, options) {
```
Comment (masteralx) [inline src/stores/DialogStore/DialogItemModel.js:20]: describe params

**Review by masteralx — CHANGES_REQUESTED** [review]

File: src/stores/DialogStore/DialogStore.js (hunk 7 above)
Comment (PUndef) [reply src/stores/DialogStore/DialogStore.js:63]: upd

File: src/stores/DialogStore/DialogItemModel.js (hunk 8 above)
Comment (PUndef) [reply src/stores/DialogStore/DialogItemModel.js:20]: upd

**Review by masteralx — APPROVED** [review]

**Pull request merged**